crates/ruma-common/src/events/poll.rs

```diff
@@ -23,6 +22,14 @@ pub mod unstable_end;
 pub mod unstable_response;
 pub mod unstable_start;
 
+/// The data from a poll response necessary to compile poll results.
+#[derive(Debug, Clone, Copy)]
+pub struct PollResponseData<'a> {
```

Comment:
Can fields of this struct be pub?
Reason is one must be able to create valid instances of it in order to invoke `compile_unstable_poll_results()`.
`OriginalSyncUnstablePollResponseEvent.data()` isn't useful in this context because inside the `event_handler` module only the EventContent is available.

Reply:
Done